### PR TITLE
PLAT-169551: Updating connection spec IDs for batch destinations and other related updates after cloud storage destinations GA [REVIEW]

### PR DESCRIPTION
**Pull request by vgiurgiu**
Updates for PLAT-169551 and PLAT-172040

### CODE REVIEW

File: src/swagger-specs/destinations.yaml [hunk 1]
```diff
@@ -48,7 +49,7 @@ tags:
...
+      <tr><td>Source connection</td><td>Source connections create and manage a connection to the internal source from where data is exported. For destination connections, the source connection can be either the <a href="https://experienceleague.adobe.com/docs/experience-platform/landing/license/data-management-best-practices.html#understanding-adobe-experience-platform-data-storage">Experience Platform Profile Store</a> when exporting segments or <a href="https://experienceleague.adobe.com/docs/experience-platform/landing/license/data-management-best-practices.html#understanding-adobe-experience-platform-data-storage">Experience Platform Data Lake</a>, when [exporting datasets](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/activate/export-datasets.html?lang=en). </td></tr>
```
Comment (JCON3DEV) [inline src/swagger-specs/destinations.yaml:52]: ah one more language specific query param on the URL

File: src/swagger-specs/destinations.yaml [hunk 2]
```diff
@@ -973,7 +970,7 @@ paths:
         When performing listing (GET) requests in the Flow Service API, you can use query parameters to [sort and filter responses](https://experienceleague.adobe.com/docs/experience-platform/dataflows/api/sort-and-filter.html?lang=en).
```
Comment (JCON3DEV) [inline src/swagger-specs/destinations.yaml:970]: Not related to your work but i spotted another '?lang=en' maybe do a quick find and delete for the file.

File: src/swagger-specs/destinations.yaml [hunk 3]
```diff
@@ -4508,7 +4480,7 @@ components:
           id: 64ef4b8b-a6e0-41b5-9677-3805d1ee5dd0
           version: '1.0'  
 
-    Create_S3_Target_Connection_Dataset_V2:
+    Create_S3_Target_Connection_Dataset:
       summary: (Beta) Datasets - Specify data format and storage location information to export files to an Amazon S3 instance
```
Comment (JCON3DEV) [inline src/swagger-specs/destinations.yaml:4484]: Should this Beta still be here?

**Review by JCON3DEV — COMMENTED** [review]
LGTM, some very minor edits to make.

File: src/swagger-specs/destinations.yaml (hunk 2 above)
Comment (vgiurgiu) [reply src/swagger-specs/destinations.yaml:970]: yep, there were actually 20+ instances of `?lang=en` that I removed now

**Review by JCON3DEV — APPROVED** [review]
LGTM

**Pull request merged**